### PR TITLE
Color placeholder ✨

### PR DESCRIPTION
**Pull request by igorkamyshev**
Hello! It is a great library, thanks for it. We use it on our site, and I really love it.

We have one corner-case and want to new feature for handle it. Some of our images really big and loading takes a lot of time. Now, we manually set the background for any large image, and users see the placeholder-color for fallback.

I suggest to add this feature to lozad. 🧑‍💻

Please, review my idea and PR and give me feedback.

I attach demo page with this feature and reeeeeally slow internet imitation.

ezgif com-video-to-gif

### CODE REVIEW

**Comment by Uncleseneca** [comment]
Cool feature!
Can't wait for it!

**Comment by AlexandrHoroshih** [comment]
This actually would be pretty useful feature!

Suggestion: It would be great to use `background` property instead of `background-color` or add similar api for `background-image`. Then it will be possible to use gradients as a placeholders. I think that good gradient might look cooler than just a color

**Comment by igorkamyshev** [comment]
@AlexandrHoroshih sure, great idea. Done.

**Comment by igorkamyshev** [comment]
@ApoorvSaxena friendly ping 👻

File: README.md [hunk 1]
```diff
@@ -202,6 +202,16 @@ const coolImage = document.querySelector('.image-to-load-first')
 observer.triggerLoad(coolImage);
 ```
 
+### Large image improvment
+
+Sometimes image loading takes a long time. For this case, you can add a placeholder color:
```
Comment (ApoorvSaxena) [inline README.md:207]: `you can add a placeholder background`

File: README.md [hunk 2]
```diff
@@ -202,6 +202,16 @@ const coolImage = document.querySelector('.image-to-load-first')
...
+
+```html
+<img class="lozad" data-placeholder="red" data-src="image.png" />
+```
+
+Lozad set the background of `img` element and users will see the fallback.
```
Comment (ApoorvSaxena) [inline README.md:213]: Set a placeholder background color of `img` element and users will see the fallback till the image loads.

File: demo/index.html [hunk 3]
```diff
@@ -58,7 +58,7 @@ <h1><a href="https://github.com/ApoorvSaxena/lozad.js"><strong>Lozad.js</strong>
                 <h1>Lozad.js - Performant Lazy Loading Library</h1>
             </article>
             <article class="thumb">
-                <img class="lozad" data-src="images/thumbs/01.jpg" data-index="1" />
+                <img class="lozad" data-placeholder="#583624" data-src="images/thumbs/01.jpg" data-index="1" />
                 <noscript><img src="images/thumbs/01.jpg" data-index="1" /></noscript>
```
Comment (ApoorvSaxena) [inline demo/index.html:62]: @igorkamyshev will be great to have this placeholder color added in all images of demo page 👍

**Comment by ApoorvSaxena** [comment]
@igorkamyshev will be great to have adjoining test cases accompanied with PR

**Comment by ApoorvSaxena** [comment]
@igorkamyshev looking forward to merge this PR soon 😃

**Comment by igorkamyshev** [comment]
@ApoorvSaxena hi, thanks for answer. Sorry for long delay. Now I fix all issues. Please, review it again and give me your feedback.

**Pull request merged**